src/OpenDebugAD7/AD7DebugSession.cs

```
@@ -1667,6 +1693,49 @@ protected override void HandleModulesRequestAsync(IRequestResponder<ModulesArgum
             responder.SetResponse(response);
         }
 
+        // test -- need to delete
+        protected override void HandleDisassembleRequestAsync(IRequestResponder<DisassembleArguments, DisassembleResponse> responder)
+        {
+            DisassembleResponse response = new DisassembleResponse();
+
+            int hr;
+            DisassembleArguments disassembleArguments = responder.Arguments;
+            Debug.Assert(!string.IsNullOrEmpty(disassembleArguments.MemoryReference));
+            try
+            {
+                IDebugMemoryContext2 memoryContext = GetMemoryContext(disassembleArguments.MemoryReference, disassembleArguments.Offset, out ulong address, out hr);
+                if (m_program.GetDisassemblyStream(enum_DISASSEMBLY_STREAM_SCOPE.DSS_ALL, (IDebugCodeContext2)memoryContext , out IDebugDisassemblyStream2 disassemblyStream) == HRConstants.S_OK)
+                {
+                    if (disassemblyStream.Seek(enum_SEEK_START.SEEK_START_BEGIN, (IDebugCodeContext2)memoryContext, address, (long)disassembleArguments.InstructionOffset) == HRConstants.S_OK)
```

>== HRConstants.S_OK [](start = 171, length = 19)

Throw if these operations fail instead of returning nothing?